src/components/Card.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styles from '../styles/card.module.css'
+function Card(props) {
+  return (
+    <a href="/#"  className={styles.course}>
+    <img src={props.course.image} style={{width:"100%"}} alt="course"/>
+    <div className={styles.courseText}>
+        <h3>{props.course?.title??"Course Name"}</h3>
+        <span className={styles.instructorName}>{props.course?.instructor??"Jeff Bezos"}</span>
+        <div>
+            <span className={styles.rating}>{props.course?.rating??4.5}</span>
+            {props.course.rating>=1?<span className={"fa fa-star "+styles.checked}></span>:<span className="fa fa-star"></span>}
+            {props.course.rating>=2?<span className={"fa fa-star "+styles.checked}></span>:<span className="fa fa-star"></span>}
+            {props.course.rating>=3?<span className={"fa fa-star "+styles.checked}></span>:<span className="fa fa-star"></span>}
+            {props.course.rating>=4?<span className={"fa fa-star "+styles.checked}></span>:<span className="fa fa-star"></span>}
+            {props.course.rating>=5?<span className={"fa fa-star "+styles.checked}></span>:<span className="fa fa-star"></span>}
```

Comment:
Nitpick: While this is working fine, but it's obviously repetitive code, if you need to implement a 10-stars based rating this wouldn't be the way to do it, right?
You can go for a loop-based solution like this below:
```js
{Array(parseInt(rating)).fill().map((_,i)=>(
    <span key={i} className={"fa fa-star "+styles.checked} />
))}
{Array(parseInt(5 - rating) < 0 ? 0 : parseInt(5 - rating)).fill().map((_,i)=>(
    <span key={i} className="fa fa-star" />
))}
```